gnocchi/tests/functional/gabbits/aggregation.yaml

```diff
@@ -361,3 +361,13 @@ tests:
     - name: get measure aggregates with no resource name
       POST: /v1/aggregation/resource/generic/metric
       status: 405
+
+    - name: get measure aggregates with None as min in clip operation
+      POST: /v1/aggregates?fill=0&granularity=1
+      data:
+        resource_type: generic
+        search: {}
+        operations: '(clip (metric mean) None 10)'
```

Comment:
sorry, i meant we should support this case ideally,  not make it fail.

Reply:
I guess the reason it is failing because, we are forcing the value to be float.
https://github.com/gnocchixyz/gnocchi/blob/master/gnocchi/rest/aggregates/api.py#L111

Reply:
sorry for delay. yes, you're right. i imagine you'd need to look at adding something like: https://github.com/gnocchixyz/gnocchi/blob/master/gnocchi/rest/aggregates/api.py#L73-L82

i can try to take a look at this over weekend if anything.